src/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@
 # This file is subject to the license terms in the LICENSE file
 # found in the top-level directory of this distribution.
 
+
+option(CPPAST_TEMPLATE_FULLARGUMENTSPARSING "Enable full argument parsing of template instantiation" OFF)
+IF(CPPAST_TEMPLATE_FULLARGUMENTSPARSING)
```

Comment:
Nitpick, please use lower case if() and endif() (and you also don't need to repeat the condition in the endif()) for consistency. The option name can also deserve a couple more underscores :wink:

Reply:
As mentioned, I also prefer to have it enabled by default.